plugin/storage/factory.go

```diff
@@ -46,6 +46,7 @@ const (
 	badgerStorageType        = "badger"
 	downsamplingRatio        = "downsampling.ratio"
 	downsamplingHashSalt     = "downsampling.hashsalt"
+	spanStorageType          = "span-storage.type"
```

Comment:
```suggestion
	spanStorageType          = "span-storage-type"
```

and I don't think you need a constant for metric name

Reply:
actually, this constant might already be defined elsewhere